ispcrt/ispcrt.hpp

```diff
@@ -149,6 +149,7 @@ class Device : public GenericObject<ISPCRTDevice> {
     Module staticLinkModules(ISPCRTModule* modules, const uint32_t num);
     // check memory type
     ISPCRTAllocationType getMemoryAllocType(void *memBuffer);
+    ISPCRTDeviceType getDeviceType() const;
```

Comment:
Here and in all other modified files: please move this method closer to other device-related methods. It should not be mixed with memory allocation methods.